packages/gatsby-telemetry/src/event-storage.ts

```diff
@@ -1,34 +1,44 @@
-const os = require(`os`)
-const path = require(`path`)
-const Store = require(`./store`)
-const fetch = require(`node-fetch`)
-const Configstore = require(`configstore`)
-const { ensureDirSync } = require(`fs-extra`)
-
+import os from "os"
+import path from "path"
+import Configstore from "configstore"
+import createFetch from "@turist/fetch"
+import { Store } from "./store"
+import { ensureDirSync } from "fs-extra"
 import { isTruthy } from "./is-truthy"
 
+const fetch = createFetch()
+
 /* The events data collection is a spooled process that
  * buffers events to a local fs based buffer
  * which then is asynchronously flushed to the server.
- * This both increases the fault tolerancy and allows collection
+ * This both increases the fault tolerance and allows collection
  * to continue even when working offline.
  */
 module.exports = class EventStorage {
```

Comment:
Is this module an external export? Can we switch to a named export with breaking stuff? It would be a major bump if it's exported, but in general if we can switch to ES module exports when doing a TS port, it's best to.

Reply:
This is still being used in files that aren't converted to typescript, I think those would break?

Reply:
They could be converted to work: `const { EventStorage } = require("./event-storage")` works with JS, but so do ESM imports.

Reply:
sweet, thanks!